tests/testthat/test-ff_standings.R

```diff
@@ -18,9 +18,13 @@ with_mock_api({
     got_schedule <- ff_schedule(got_conn, week = 4)
     got_potentialpoints <- .flea_add_potentialpoints(got_schedule, got_conn)
 
+    tony_conn <- espn_connect(season = 2020, league_id = 899513)
+    tony_standings <- ff_standings(tony_conn)
```

Comment:
Do these tests run/work? :)

Reply:
so, i ran `devtools::check()` and was getting an issue with the sleeper `ff_standings()` call, unless i was misreading the output and it was really an issue with this new ESPN `ff_standings()`. but when i ran `testthat::test_local()` every test passed

Reply:
Looking at it rn, brb

Reply:
LGTM! Standings should theoretically incorporate potential points (aka optimal lineups) to be consistent with other methods, I do have some code for it that we could probably replicate once ff_schedule code is finalized

Reply:
oh, so is that is what the potential points are for lol... yeah we can do that. just wanted to get the meat of the function done

Reply:
Can you rerun the test output on devtools check and screenshot the bug to me?

Reply:
i said sleeper earlier but it was actually fleaflicker

![Capture](https://user-images.githubusercontent.com/15663460/109435332-9df9e500-79df-11eb-955b-87014ef8fac4.PNG)